utils/create-keying-sheet-tfm-facilities/database-client/mongo-db-client.ts

```diff
@@ -0,0 +1,17 @@
+import z from 'zod';
+import { MongoDbClient } from '@ukef/dtfs2-common/mongo-db-client';
+
+const mongoDbConfigSchema = z.object({
+  MONGODB_URI: z.string(),
+  MONGO_INITDB_DATABASE: z.string(),
+});
+
+const getMongoDbClient = (): MongoDbClient => {
+  const mongoDbConfig = mongoDbConfigSchema.parse(process.env);
+  return new MongoDbClient({
+    dbName: mongoDbConfig.MONGO_INITDB_DATABASE,
+    dbConnectionString: mongoDbConfig.MONGODB_URI,
```

Comment:
I might be confused, but why does this not use `MONGODB_URI_QA`?

Reply:
The reason initially was that in order to run this against the feature DB I was updating the `MONGODB_URI` env variable and not realising that the db client was actually using the `MONGODB_URI_QA` variable. However, I think it makes more sense to keep things consistent and use `MONGODB_URI_QA` here so I'll update that